OgreMain/include/OgreString.h

```diff
@@ -55,6 +55,24 @@ THE SOFTWARE.
 #   define strtoll_l(ptr, end, base, l) strtoll(ptr, end, base)
 #endif
 
+#if OGRE_PLATFORM == OGRE_PLATFORM_LINUX
+#if !defined(HAVE_STRTOUL_L)
```

Comment:
this will break for projects consuming OGRE, as we do not make sure `HAVE_STRTO*` is defined for them. Also macros in the public header must be namespaced i.e. start with `OGRE_*`.

Reply:
OK done